application/src/test/resources/GetCaseTypesResponseForCCD_TestDefinition_V45.json

```diff
@@ -13,6 +13,10 @@
     "description" : "Content for the Test Jurisdiction.",
```

Comment:
would it be possible to properly model the banner in Object Oriented terms with a Banner object?

Reply:
We can do that, and need to change database structure as well, need to introduce new table for Banner and reference jurisdiction id in the new table..

Reply:
@kiran-yenigala-hmcts I'd recommend we do that. That we introduce a banner table. I really don't like introducing banner info into the jur table. I would also have a separate Banner worksheet in the spreadsheet dedicated to the banner. Was having a chat with @rafalkalita and he has my same concerns

Reply:
basically I'd like to see ideally 0 changes on the Jur code

Reply:
Done changes as requested